README.md

```diff
@@ -15,6 +15,7 @@ To add a conference to the list, please [contribute](#contributing)!
 | [DevConf.IN](https://devconf.info/in) | 2-3 Aug | Bengaluru, India | DevConf.IN 2019 is the second annual Developers' Conference to be organised by Red Hat in India. | No (Free for all) |
 | [AWS Community Day, Bangalore](https://communityday.awsugblr.in/) | 26-27 Jul | Conrad, Bangalore | Flagship community event organized by AWS UserGroup that's spread over 2 days including workshops and the conference. | No |
 | [The Fifth Elephant](https://hasgeek.com/fifthelephant/2019/) | 25-26 Jul | NIMHANS Convention Centre, Bangalore | The eighth edition of India's best conference on data. | Yes, for speakers |
+| [MEC.Conf](https://conf.mec.dev/) | 26-28 Jul | Kerala Technology Innovation Zone, Kochi | MEC.conf, the flagship conference organized by IEEE, IETE and FOSS-MEC of Govt Model engineering college,Kochi. | No |
```

Comment:
@kurianbenoy Thanks for adding the conf! :)

The list is in the reverse chronological order, can you move this above the AWS Community Day conf? Looks good otherwise. :+1:

Reply:
@kurianbenoy, you might find [this PR](https://github.com/kurianbenoy/tech-conferences-india/pull/1) helpful...